runtime/src/lib.rs

```diff
@@ -70,12 +72,22 @@ pub use content_directory::{
     HashedTextMaxLength, InputValidationLengthConstraint, MaxNumber, TextMaxLength, VecMaxLength,
 };
 
+#[cfg(feature = "std")]
+/// Wasm binary unwrapped. If built with `BUILD_DUMMY_WASM_BINARY`, the function panics.
+pub fn wasm_binary_unwrap() -> &'static [u8] {
+    WASM_BINARY.expect(
+        "Development wasm binary is not available. This means the client is \
+        built with `BUILD_DUMMY_WASM_BINARY` flag and it is only usable for \
+        production chains. Please rebuild with the flag disabled.",
+    )
+}
+
 /// This runtime version.
 pub const VERSION: RuntimeVersion = RuntimeVersion {
     spec_name: create_runtime_str!("joystream-node"),
     impl_name: create_runtime_str!("joystream-node"),
-    authoring_version: 7,
-    spec_version: 7,
+    authoring_version: 8,
```

Comment:
I think it is rather risky to have it as a runtime upgrade. Because we cannot guarantee (neither can Parity) that 2.0.0rc4 libs are fully compatible with 2.0.0. The worst-case scenario would be "it works most of the time but fails on rare occasions".

Reply:
Doing some testing here https://github.com/Joystream/joystream/pull/1717

Reply:
I remember we discussed this briefly, I agree since we don't have 100% test coverage in the integration tests there is a risk. However didn't polkadot and kusama do runtime upgrades recently to latest version? Can we ask or look into how they managed it.

Reply:
So far my testing shows that it will be possible to upgrade the chain.
- The native binary can sync the current chain. So in terms of consensus and networking it has no issues.

![Screen Shot 2020-11-16 at 5 23 51 PM](https://user-images.githubusercontent.com/1621012/99260981-871af280-2835-11eb-93a0-46c82c7dcc11.png)

- I ran a dev babylon chain initialized it with some state and did a runtime upgrade (kept the same authoring version and bumped the spec version) without issues.. block production, reading state, executing transactions all works as expected. (Using hosted polkadot-js/apps website for a working "pioneer")